examples/type-test-expression/type_test_expression.out

```diff
@@ -6,3 +6,5 @@ Is 'a' a string? true
 Alex is a student
 Alex is a person
 Alex is not a vehicle
+Is foo returns a student?true
```

Comment:
```suggestion
Does foo return a student? true
```

Reply:
Fixed .bal and .out file to show suggested output.